dbms/src/main/java/org/polypheny/db/ddl/DdlManagerImpl.java

```diff
@@ -313,6 +319,9 @@ public void dropAdapter( String name, Statement statement ) {
                         catalog.getAllocRel( defaultNamespaceId ).deleteColumn( allocation.id, column.columnId );
                     }
 
```

Comment:
This assumes that source adapters always add their entities to the default namespace?

Reply:
Polypheny always just put Sources into the default namespace, I just replaced the previously unspecific numbers with the correct constants.
I am also aware that this is rather limiting while the deploying sources, but we neither have the language nor the visual components to allow for a satisfiable configuration during this process yet.